pkg/client/client.go

```diff
@@ -33,4 +33,7 @@ type Client interface {
 	GetImages() (*models.Images, error)
 	GetNetworks() (*models.Networks, error)
 	GetCloudServiceInstances() ([]bluemixmodels.ServiceInstanceV2, error)
+
+	GetZone() string
+	GetRegion() string
```

Comment:
do we really need to add these 2 functions too? I'm just wondering what advantages we are going go get by adding them here in the interface?!